.gitattributes

```diff
@@ -0,0 +1,8 @@
+# Force linux line ending for all as "text" file recognized files.
```

Comment:
This might be useful, but before introducing such a thing I'd prefer to actually have a problem to fix. As far as I know this fine very rarely exist in open source project, which let me thing that git must have sensible default.

Reply:
I understand. To be honest, I only introduced this after I noticed problems with the container setup on windows.
Prettier complained about wrong line endings. Forcing a common linefeed wasn't my preferred solution, but nothing else seemed to work properly.